app/views/admin/responses/_attachments.html.erb

```diff
@@ -0,0 +1,38 @@
+<% if attachable.can_order_attachments? && attachable.attachments.many? %>
```

Comment:
Yes, that would be better. I think the only problem is that we don't need the heading in the `attachments/attachments` partial in this instance. I can't see if there's a way to optionally exclude this, otherwise I guess we would need to change the other view that calls this to include the heading at that point.

Reply:
Oh right i see. Yea i think we can pull the heading into the views above like you said. The other option is to pass something into the locales but i think that's probs a bit overkill